templates/cloudformation/aws-resources.yaml

```diff
@@ -15,6 +19,12 @@ Parameters:
     Default: 'change-me'
     Description: AEM License for Deployment
     Type: String
+  AEMKeyStorePasswordParameter:
+    Description: Java Keystore password used in AEM Author and Publish
+    Type: String
```

Comment:
Please add a TODO note here about moving to secure string when it's supported. Also please add a link that explains why secure string cannot currently be used.